optimism/eth_getBlockByNumber.yaml

```diff
@@ -31,3 +31,86 @@ paths:
               const alchemy = new Alchemy(settings);
 
               alchemy.core.getBlock(15221026).then(console.log)
+      responses:
+        '200':
+          description: 'Returns a block object with the following fields, or null when no block was found.'
+          content:
+            application/json:
+              schema:
+                allOf:
```

Comment:
Note for future - the reason why we did this was because optimism has a slightly different payload that other chains